INSTALL_DOCKER.rst

```diff
@@ -0,0 +1,259 @@
+
+  ::
+
+    sudo swupd bundle-remove containers-basic
+
+Install Clear Containers bundles
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. attention::
+   Once appropriate bundles become availabel on Clear Linux the below installation commands will be updated.
+
+Install the following bundles and RPMs to enable our work in progress linux-container-testing packages.
+
+  ::
+
+    sudo swupd bundle-add os-clr-on-clr
+    sudo swupd bundle-add os-core-dev
+    sudo swupd bundle-add os-dev-extras
+    sudo rpm -ivh --nodeps --force https://download.clearlinux.org/current/x86_64/os/Packages/qemu-lite-bin-2.6.0-17.x86_64.rpm
+    sudo rpm -ivh --nodeps --force https://download.clearlinux.org/current/x86_64/os/Packages/qemu-lite-data-2.6.0-17.x86_64.rpm
+    sudo rpm -ivh --nodeps --force https://download.clearlinux.org/current/x86_64/os/Packages/json-glib-dev-1.2.0-8.x86_64.rpm
+    sudo rpm -ivh --nodeps --force https://download.clearlinux.org/current/x86_64/os/Packages/json-glib-lib-1.2.0-8.x86_64.rpm
+    sudo rpm -ivh --nodeps --force https://download.clearlinux.org/current/x86_64/os/Packages/linux-container-testing-4.5-9.x86_64.rpm
+    sudo rpm -ivh --nodeps --force https://download.clearlinux.org/current/x86_64/os/Packages/linux-container-testing-extra-4.5-9.x86_64.rpm
+    #Note: Ignore the errorldconfig:*
+    #/usr/lib64/libguile-2.0.so.22.7.2-gdb.scm is not an ELF file - it has the wrong magic bytes at the start.*
+
+Check QEMU-lite
+~~~~~~~~~~~~~~~
+
+`Clear Containers`_ uses an optimised version of `QEMU`_ called `QEMU-lite`_
+You can now check that the `QEMU-lite`_ package is installed and functioning:
+
+  ::
+
+    # qemu-lite-system-x86_64 --version
+    QEMU emulator version 2.6.0, Copyright (c) 2003-2008 Fabrice Bellard
+
+    # qemu-lite-system-x86_64 --machine help | grep pc-lite
+    pc-lite Light weight PC (alias of pc-lite-2.6)
+
+    pc-lite-2.6Light weight PC
+
+Install the Docker bundle
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We can now install the `Clear Linux`_ bundle that containers Docker_:
+
+  ::
+
+    sudo swupd bundle-add opencontainers-dev
+
+Add your user to the KVM and Docker groups
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To enable your user to access both Docker and KVM you will need to add them to the relevant groups on the machine:
+ 
+  ::
+
+    sudo usermod -G kvm,docker -a <USERNAME>
+
+Download the clr-oci-runtime 2.0 code
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. attention::
+   Update this section when clr-oci-runtime is available in a Clear Linux bundle
+
+Download, build and install the clr-oci-runtime from source:
+
+  ::
+
+    cor_source=${HOME}/clr-oci-runtime
+    git clone https://github.com/01org/clr-oci-runtime.git $cor_source
+    cd $cor_source
+    autoreconf -fvi
+    bash autogen.sh --disable-cppcheck --disable-valgrind
```

Comment:
I think we leave it as is - and then we'll have to work out the clash for the users who have installed from source.